src/main/java/no/sikt/nva/scrapers/EntityDescriptionExtractor.java

```diff
@@ -94,6 +114,11 @@ private static PublicationDate extractPublicationDate(DublinCore dublinCore) {
         return new PublicationDate(date, publicationDateNva);
     }
 
+    @SuppressWarnings("PMD.InefficientEmptyStringCheck")
```

Comment:
https://pmd.github.io/latest/pmd_rules_java_performance.html#inefficientemptystringcheck
There is a reason for this warning, but as this is not production code, it is likely no big problem. We should however consider a utility function for this is nav-commons if it does not already exist.